R/covr.R

```diff
@@ -44,7 +58,10 @@ trace_calls <- function (x, srcref = NULL) {
       fun_body <- trace_calls(fun_body)
     }
 
-    formals(x) <- trace_calls(formals(x))
+    #formals(x) <- trace_calls(formals(x))
```

Comment:
What edge case causes the need for this extra logic?  Do you have a test for it?